addons/web/static/src/js/fields/basic_fields.js

```diff
@@ -1966,6 +2002,17 @@ var KanbanFieldBinaryImage = FieldBinaryImage.extend({
     events: {},
 });
 
+var KanbanCharImageUrl = CharImageUrl.extend({
+    // In kanban views, there is a weird logic to determine whether or not a
+    // click on a card should open the record in a form view.  This logic checks
+    // if the clicked element has click handlers bound on it, and if so, does
+    // not open the record (assuming that the click will be handle by someone
```

Comment:
"handle" -> "handled"

Reply:
It's another c/c thing which I'm not sure is even needed but I fixed the typo :D